tools/set_version.py

```diff
@@ -48,9 +48,9 @@ def substitute_file(name):
         elif 'VERSION=' in line and name.endswith('build_dist.sh'):
             line = 'VERSION=%d.%d.%d\n' % (version[0], version[1], version[2])
         elif 'version=' in line and name.endswith('setup.py'):
-            line = "    version='%d.%d.%d',\n" % (version[0], version[1], version[2])
+            line = "    version=\"%d.%d.%d\",\n" % (version[0], version[1], version[2])
         elif "version = '" in line and name.endswith('setup.py'):
-            line = "    version='%d.%d.%d',\n" % (version[0], version[1], version[2])
+            line = "    version=\"%d.%d.%d\",\n" % (version[0], version[1], version[2])
```

Comment:
it looks like this line is duplicated. I should remove one of them once this lands